internal/pipe/pipeline_test.go

```diff
@@ -179,6 +184,10 @@ func TestPipelineTwoCommandsPiping(t *testing.T) {
 }
 
 func TestPipelineDir(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("FIXME: test skipped on Windows: 'pwd' incompatibility")
```

Comment:
If you can assume that this is run in the MSYS2 Bash that is also used by Git for Windows, `pwd -W` should give you what you want on Windows.

Reply:
Hmmm, that [doesn't seem to have worked](https://github.com/github/git-sizer/runs/4168305310?check_suite_focus=true). I guess we're talking about another bash here? :shrug:

Reply:
OK, I found a substitute for `pwd` for Windows (implemented in c991c8e0c8436a2547ef7cdfd48a554cec0eecee):

    "bash", "-C", "pwd -W"

Presumably `pwd` is implemented as a `bash` builtin rather than a separate executable. Plus, some directory name cleaning is needed, since that command returns a result that uses forward slashes whereas the Go functions' results use backslashes.

Reply:
> Hmmm, that [doesn't seem to have worked](https://github.com/github/git-sizer/runs/4168305310?check_suite_focus=true). I guess we're talking about another bash here? 🤷

That's funny. It _does_ use the expected Bash. I guess that Go is _smart_ and uses CMD. And even if the MSYS2 `pwd.exe` is in the `PATH`, it does not work because in contrast to the Bash built-in, it does _not_ support `-W`:

```console
$ /c/Program\ Files/Git/usr/bin/pwd.exe  -W
pwd: unknown option -- W
Try '/c/Program Files/Git/usr/bin/pwd --help' for more information.
```

So: sorry for my unhelpful suggestion.

Reply:
NP; your suggestion of `-W` put me on the right track :sparkles: